doc/index.md

```diff
@@ -307,7 +307,7 @@ provides, and a blurb about how each is intended to be used.
 <DT>\link ranges::views::linear_distribute_fn `views::linear_distribute`\endlink</DT>
   <DD>Distributes `n` values linearly in the closed interval `[from, to]` (the end points are always included). If `from == to`, returns `n`-times `to`, and if `n == 1` it returns `to`.</DD>
 <DT>\link ranges::views::move_fn `views::move`\endlink</DT>
-  <DD>Given a source range, return a new range where each element has been has been cast to an rvalue reference.</DD>
+  <DD>Given a source range, return a new range where each element has been cast to an rvalue reference.</DD>
```

Comment:
```suggestion
  <DD>Given a source range, return a new range where each element has been casted to an rvalue reference.</DD>
```

Reply:
The past tense of cast is actually cast itself, not casted 🤣